### PR TITLE
Changed default location in New-AzVM simple parameter set

### PR DESCRIPTION
**Pull request by niander**
## Description

Issue #8466 

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1 [hunk 1]
```diff
@@ -452,11 +452,13 @@ function Test-SimpleNewVmImageName
...
         # Common
-        $x = New-AzVM `
+        $imgversion = Get-VMImageVersion -publisher "MicrosoftWindowsServer" `
+							-offer "WindowsServer" -sku "2016-Datacenter"
+		$x = New-AzVM `
             -Name $vmname `
             -Credential $cred `
             -DomainNameLabel $domainNameLabel `
-            -ImageName "MicrosoftWindowsServer:WindowsServer:2016-Datacenter:2016.127.20170406"
+            -ImageName ("MicrosoftWindowsServer:WindowsServer:2016-Datacenter:" + $imgversion)
```
Comment (markcowl) [inline src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1]: Can't we just use 'latest' here as the image version?

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1 [hunk 2]
```diff
@@ -105,7 +105,7 @@ function Test-SimpleNewVmWithUltraSSD
 		#As of now the ultrasd feature is only supported in east us 2 and in the size Standard_D2s_v3, on the features GA the restriction will be lifted
 		#Use the follwing command to figure out the one to use 
 		#Get-AzComputeResourceSku | where {$_.ResourceType -eq "disks" -and $_.Name -eq "UltraSSD_LRS" }
-        $x = New-AzVM -Name $vmname -Credential $cred -DomainNameLabel $domainNameLabel -Location "east US 2" -EnableUltraSSD -Zone 3 -Size "Standard_D2s_v3"
+        $x = New-AzVM -Name $vmname -Credential $cred -DomainNameLabel $domainNameLabel -Location "eastus2" -EnableUltraSSD -Zone 2 -Size "Standard_D2s_v3"
```
Comment (markcowl) [inline src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1]: Have we run this test against mooncake / government cloud?

Comment (niander) [reply src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1]: Are you talking specifically about the UltraSSD?

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1 (hunk 1 above)
Comment (niander) [reply src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1]: Yes. Another test is using latest

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1 (hunk 2 above)
Comment (niander) [reply src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1]: Yes.

**Pull request merged**